src/main/java/com/synerzip/projectmanagementapp/exception/MyExceptionMapper.java

```diff
@@ -0,0 +1,38 @@
+package com.synerzip.projectmanagementapp.exception;
+
+import java.nio.channels.OverlappingFileLockException;
+
+import javax.persistence.EntityNotFoundException;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.hibernate.ObjectNotFoundException;
+
+import com.synerzip.projectmanagementapp.model.ErrorMessage;
+
+@Provider
+public class MyExceptionMapper implements ExceptionMapper<Exception> {
```

Comment:
should not have these kind of class name. (MY ??)
Class should have proper nomenclature.